docs/content/topics/project-developers/projects.md

```diff
@@ -9,296 +9,267 @@ aliases:
 
-If you have the `brig` client installed, you can use it to list and interact with
-your projects:
+For example, if project secrets are contained in a `secrets.yaml` file, the
+private key would be added like so:
 
-```
-$ brig project list
-NAME                       	ID                                                            	REPO
-technosophos/brigade-trello	brigade-635e505c74ad679bb9144d19950504fbe86b136ac3770bcff51ac6	github.com/technosophos/brigade-trello
-brigadecore/empty-testbed         	brigade-830c16d4aaf6f5490937ad719afd8490a5bcbef064d397411043ac	github.com/brigadecore/empty-testbed
-technosophos/hello-helm    	brigade-b140dc50d4eb9136dccab7225e8fbc9c0f5e17e19aede9d3566c0f	github.com/technosophos/hello-helm
-technosophos/twitter-t     	brigade-cf0858d449971e79083aacddc565450b8bf65a2b9f5d66ea76fdb4	github.com/technosophos/twitter-t
+```yaml
+gitSSHKey: |-
+  -----BEGIN RSA PRIVATE KEY-----
+  IIEpAIBAAKCAg1wyZD164xNLrANjRrcsbieLwHJ6fKD3LC19E...
+  ...
+  ...
+  -----END RSA PRIVATE KEY-----
 ```
 
-You can also directly inspect your project with `brig`:
+The project secrets can then be updated via the usual brig command:
 
 ```console
-$ brig project get brigadecore/empty-testbed
-id: brigade-830c16d4aaf6f5490937ad719afd8490a5bcbef064d397411043ac
-name: brigadecore/empty-testbed
-repo:
-  name: github.com/brigadecore/empty-testbed
-  owner: ""
-  cloneurl: https://github.com/brigadecore/empty-testbed.git
-  sshkey: ""
-kubernetes:
-  namespace: default
-  vcssidecar: brigadecore/git-sidecar:latest
-  buildStorageSize: "50Mi"
-sharedsecret: FakeSharedSecret
-github:
-  token: 76faketoken789
-secrets:
-  dbPassword: supersecret
+$ brig project secrets set --file secrets.yaml
 ```
 
+### Using other Git providers
 
-## Internal Brigade Project Names
-
-Brigade creates an "internal name" for each project. It looks something like
-this:
-
-```
-brigade-635e505c74ad679bb9144d19950504fbe86b136ac3770bcff51ac6
-```
-
-There is nothing fancy about this name. In fact, it is just a prefixed hash of
-the project name. Its purpose is merely to ensure that we can meet the naming
-requirements of Kubernetes without imposing undue restrictions on how you name
-your project.
-
-_These names are intentionally repeatable._ Two projects with the same name should
-also have the same internal name.
-
-## Using SSH Keys
-
-You can use SSH keys and a `git+ssh` URL to secure a private repository.
-
-In this case, your project's `cloneURL` should be of the form `git@github.com:brigadecore/brigade.git`
-and you will need to add the SSH _private key_ to the `values.yaml` file.
-
-When doing `brig project create`, URLs that do not use HTTP or HTTPS will prompt
-for (optionally) adding an SSH key.
-
-## Using other Git providers
+Brigade currently has gateway support for [GitHub] and [BitBucket]. Other
+providers should work fine for Brigade projects, though gateways to handle
+webhooks might not yet exist. See the [Gateways] doc for further info.
```

Comment:
I'm worried about the potential for this section to be confusing.

I know you know that Brigade just has generalized git support and works with any repo from any provider and the gateways play no role other than as the source of events, but to the uninitiated, I'm worried that distinction doesn't come through clearly here.

Reply:
Good point.  Revised in https://github.com/brigadecore/brigade/pull/1556/commits/0175e99d9ef2b57adef2cfe37ca3b0a9f4c09d26.  PTAL when convenient.